packages/vite/src/node/plugins/importMetaGlob.ts

```diff
@@ -168,29 +169,40 @@ export async function parseImportGlob(
     if (ast.arguments.length < 1 || ast.arguments.length > 2)
       throw err(`Expected 1-2 arguments, but got ${ast.arguments.length}`)
 
-    const arg1 = ast.arguments[0] as ArrayExpression | Literal
+    const arg1 = ast.arguments[0] as ArrayExpression | Literal | TemplateLiteral
     const arg2 = ast.arguments[1] as Node | undefined
 
     const globs: string[] = []
-    if (arg1.type === 'ArrayExpression') {
-      for (const element of arg1.elements) {
-        if (!element) continue
-        if (element.type !== 'Literal') throw err('Could only use literals')
+
+    const validateLiteral = (
+      element: Literal | TemplateLiteral | ArrayExpression['elements'][0]
+    ) => {
+      if (!element) return
```

Comment:
```suggestion
    const validateLiteral = (
      element: Expression | SpreadElement
    ) => {
```
`ArrayExpression['elements'][0]` is `Expression | SpreadElement | null` and that includes `Literal | TemplateLiteral`. So I think we could simplify this.